Tiltfile

```diff
@@ -15,10 +17,11 @@ cluster = os.getenv('CLUSTER', "")
 chartmuseum_basic_auth = os.getenv('CHARTMUSEUM_BASIC_AUTH', "")
 
 # build_with options:
-# cargo: No crosscompilation, faster than cross
-# cross: Supports crosscompilaton
-build_with = os.getenv('BUILD_WITH','cross')
+# cargo: No crosscompilation, faster than cargo-zigbuild
+# zig: Supports crosscompilaton
+build_with = os.getenv('BUILD_WITH','zig')
```

Comment:
perhaps we could go just with zig i guess and simplify this , but we can do it in another PR

Reply:
Done in https://github.com/newrelic/newrelic-agent-control/pull/1511/commits/3011912f72ed6290360a7372530c8a88eebc95f7!